metadata-ingestion/src/datahub/ingestion/source/sql_common.py

```diff
@@ -44,13 +45,19 @@ def report_dropped(self, table_name: str) -> None:
 
 
 class SQLAlchemyConfig(ConfigModel):
-    options: Optional[dict] = {}
+    options: dict = {}
     table_pattern: AllowDenyPattern = AllowDenyPattern.allow_all()
 
     @abstractmethod
     def get_sql_alchemy_url(self):
         pass
 
+    def get_identifier(self, schema: str, table: str) -> str:
+        return f"{schema}.{table}"
+
+    def mangle_schema_table_names(self, schema: str, table: str) -> Tuple[str, str]:
```

Comment:
A different name like standardize_schema_table_names might be better. 
With a comment describing why we need this method.

Reply:
updated